src/utils/isSet.ts

```diff
@@ -1,6 +1,4 @@
-import { values } from 'lodash';
 
-
-export function isSet(value?: string | undefined): boolean {
-  return (typeof value === 'string' && values.length > 0);
-}
\ No newline at end of file
+export function isNonEmptyString(value?: unknown): value is string {
```

Comment:
rename the file from ~~`isSet.ts`~~ to `isNonEmptyString.ts`, so it matches the implementation and doesn't confuse people later.